.github/workflows/test-on-comment.yaml

```diff
@@ -0,0 +1,62 @@
+name: Kruize Remote monitoring functional tests
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  functest:
+
+    # Only run this if the user asked us to
+    if: github.event.comment.body == 'test'
```

Comment:
We need to be more descriptive about what `test` is going to run here. Is this release test, functional, stress etc Suggest you add a more descriptive name here